docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -722,6 +722,176 @@ directly or used in conditional statements.
 * ``$file_size``
     Size of the file for this row.
 
+Functions
+---------
+
+The connector exposes the following table functions:
+
+system.table_changes
+^^^^^^^^^^^^^^^^^^^^
+
+Allows reading Change Data Feed (CDF) entries to
+expose row-level changes between two versions of a Delta Lake table.
+When the ``change_data_feed_enabled`` table property is set to ``true``
+on a specific Delta Lake table, the connector records change events for
+all data changes on the table.
+In addition to columns present in the table this function
+the function returns the following values for each change event:
+
+* ``_change_type``
+    Type ``VARCHAR``, gives the type of change that occurred,
```

Comment:
```suggestion
    Gives the type of change that occurred as represented as ``VARCHAR`` value. 
```

Reply:
I will change it but tbh all three descriptions of these additional columns that you propose seems unnaturally long and weird to me. I tried to be as short and as precise as possible. https://docs.databricks.com/delta/delta-change-data-feed.html#what-is-the-schema-for-the-change-data-feed - this is how databricks describes them